pubtools/_quay/push_docker.py

```diff
@@ -507,6 +488,57 @@ def remove_old_signatures(
                     key,
                 )
 
+    def _fetch_digest(self, repo, tag, media_type):
```

Comment:
I'm questioning what exactly does this method accomplish. It the main purpose to combine repo + tag together into a full reference? If so, I don't believe this justifies a new method and can be done in fetch_missing_push_items_digests

Reply:
What?

Reply:
now it's called on two places in fetch_missing_push_items_digests